### PR TITLE
Update user login.  Refers to #315

### PR DESCRIPTION
**Pull request by punkael**

### CODE REVIEW

**Comment by andrewbiang888** [comment]
@punkael, is this ready for review?

**Comment by punkael** [comment]
Yes!

**Review by andrewbiang888 — APPROVED** [review]
Looks ready for me

File: app/src/components/Forms/SignIn/index.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ import SignInForm from "./SignInForm";
 /** @jsx jsx */
 import { css, jsx } from "@emotion/core";
 import { NavLink } from "react-router-dom";
+import {login } from "../../../state/ducks/auth";
+//import { connect } from "react-redux";
```
Comment (jaronheard) [inline app/src/components/Forms/SignIn/index.js]: Delete this comment

File: app/src/components/Forms/SignIn/index.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ import SignInForm from "./SignInForm";
 /** @jsx jsx */
 import { css, jsx } from "@emotion/core";
 import { NavLink } from "react-router-dom";
+import {login } from "../../../state/ducks/auth";
```
Comment (jaronheard) [inline app/src/components/Forms/SignIn/index.js]:
```suggestion
import { login } from "../../../state/ducks/auth";
```
We must not have prettier in this project ;)

**Review by jaronheard — APPROVED** [review]
There's an extra comment that could be removed, otherwise looks good

**Pull request merged**